ArduCopter/switches.cpp

```diff
@@ -550,6 +550,7 @@ void Copter::do_aux_switch_function(int8_t ch_function, uint8_t ch_flag)
             break;
 
         case AUXSW_PRECISION_LOITER:
+#if PRECISION_LANDING == ENABLED
```

Comment:
Re the case statements, I think we might get compiler warnings if we don't include every item in the enum.. I think that may be why Peter has left it in.